src/main/java/com/example/dog/dao/JdbcDogDAO.java

```diff
@@ -124,4 +111,8 @@ public Dog remove(int id) {
     private static ZonedDateTime getDateTime(Timestamp timestamp) {
         return timestamp != null ? ZonedDateTime.ofInstant(timestamp.toInstant(), ZoneId.systemDefault()) : null;
     }
+
+    private String getDateString(ZonedDateTime time) {
+        return time == null ? null : "'" + time.format(DateTimeFormatter.ofPattern("uuuu-MM-dd HH:mm:ss VV")) + "'";
```

Comment:
You can make a constant out of `DateTimeFormatter.ofPattern("uuuu-MM-dd HH:mm:ss VV")`*, so that you don't have to parse the pattern on each method invocation.

*this new `DateTimeFormatter` is thread-safe as opposed to the old one that worked with `Date`, so it's safe to make a constant out of it